odoo/addons/test_performance/tests/test_performance.py

```diff
@@ -535,7 +535,7 @@ def test_create_base_with_tags(self):
                 'tag_ids': [Command.set([])],
             })
 
-        with self.assertQueryCount(2):
+        with self.assertQueryCount(3):
```

Comment:
This change is not acceptable. It looks like adding one query to all changes on many2many fields, and I don't see the reason for that change.  Can you explain it?

Reply:
The increased one query is becasue
before this commit:
`assertQueryCount` cannot detect the query generated by `execute_values`
after this commit:
`assertQueryCount` can detect the query generated by `execute_values`